src/main/resources/application.properties

```diff
@@ -1 +1,3 @@
-
+spring.data.mongodb.database=open_drive
+spring.data.mongodb.username=open_drive_dev
+spring.data.mongodb.password=open_drive_pass
```

Comment:
Are `open_drive_dev` and `open_drive_pass` variables which store current user and its password?
If yes then where do we have to define them (don't see them in current change)?

Reply:
Those variables are defined to connect to local database. In wiki there is small doc how to create user with password and new database. I decided to add this so everyone will have the same setup. 
After having deployment process with test,preprod and prod enviroment it will be hidden